.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Let us know about an issue with DIM
+title: ''
+labels: Bug
+assignees: ''
+
+---
+
+**What version of DIM are you using?**
+Find the version in "About DIM" from the menu.
+
+**What Browser and OS are you using?**
+It helps to know what type of browser and what version you're using.
```

Comment:
If you have "installed" DIM on desktop this is likely through Chrome (i think? whats up w safari+pwa?)

Reply:
Safari can install too.